### PR TITLE
Improve account counter (ignore junk & trash)

### PR DESCRIPTION
**Pull request by aenario**
and faster synchro
some linting & comments

### CODE REVIEW

File: server/models/mailbox.coffee [hunk 1]
```diff
@@ -487,13 +487,13 @@ class Mailbox extends cozydb.CozyModel
...
-            Message.fetchOrUpdate box, msg.mid, msg.uid, (err, result) ->
+            Message.fetchOrUpdate box, msg, (err, result) ->
                 reporter.onError err if err
                 reporter.addProgress 1
                 if result?.shouldNotif is true
                     shouldNotif = true
-                # dont stop
-                cb null
+                # loop anyway, let the DS breath
+                setTimeout (-> cb null), 50
```
Comment (frankrousseau) [inline server/models/mailbox.coffee]: Isn't it the same to run `cb()` (instead of `cb null`)?

**Comment by frankrousseau** [comment]
Thank you @aenario

**Comment by frankrousseau** [comment]
The counter works well for my inbox. But the overall counter is still broken.

**Comment by frankrousseau** [comment]
I think we should simply display no counter on the mail account level.

File: server/models/mailbox.coffee (hunk 1 above)
Comment (aenario) [inline server/models/mailbox.coffee]: it is. Will fix, thanks !

**Pull request merged**